docs/getting_started/room.md

```diff
@@ -6,9 +6,13 @@ The holder for peers and components.
 
 **Optional**
 
+:::caution
+  Support for custom ID will be dropped soon in favor of IDs generated by the server.
+
 * `roomId` (string, default: null) - Custom ID used for identifying the room within Fishjam.
-  Must be unique across all rooms. If not provided, a random UUID will be generated. 
+  Must be unique across all rooms. If not provided, a random UUID will be generated.
```

Comment:
Shouldn't we inform readers that room won't work correctly if roomId is not unique across all rooms?

Reply:
Well isn't that actually stated here that: `Must be unique across all rooms` ?

Reply:
tbh: I wouldn't put much attention to it since it will be dropped soon I just added `caution` for now.

Reply:
IMO it isn't clear. It could also be mentioned that if you send the same room as on another node, you will receive an error.